include/llpc.h

```diff
@@ -66,6 +66,17 @@
 //* |     23.0 | Add flag robustBufferAccess in PipelineOptions to check out of bounds of private array.               |
 //* |     22.0 | Internal revision.                                                                                    |
 //* |     21.0 | Add stage in Pipeline shader info and struct PipelineBuildInfo to simplify pipeline dump interface.   |
+//*
+//* Important note: All structures defined in this file that are passed as input into LLPC must be zero-initialized
```

Comment:
I prefer to use uppercase to highligh it just like what we have done in source codes: NOTE, TODO

Important note -> IMPORTANT NOTE

Reply:
Yeah, that makes sense, I'm going to change it.